Chatto/sources/ChatController/Collaborators/ChatDataSourceProtocol.swift

```diff
@@ -28,6 +28,7 @@ import Foundation
 public enum UpdateType: CaseIterable {
     case normal
     case firstLoad
+    case firstSync
```

Comment:
Is the case supposed to be assigned by the client of the framework?

Reply:
Yep. We needed to differentiate it from `.normal` to perform scrolling after fetching of new messages from server is propagated to collection view